channel2/message.go

```diff
@@ -292,6 +292,35 @@ func readHelloV2(peer io.Reader) (*Message, error) {
 	return m, nil
 }
 
+func readHelloWssV2(wss_message []byte) (*Message, error) {
```

Comment:
Can you use bytes.NewReader([]byte) to wrap the messages and re-use the existing read funcs? Or is there some format difference?

Reply:
The JS SDK makes no attempt to change the Message format.

I'll give your suggestion a whirl.

Reply:
done